docs/examples/query_decomposition.md

```diff
@@ -0,0 +1,129 @@
+# Planning and Executing a Query Plan
```

Comment:
make sure to update mkdocs.yml in the root directory so you can link to it

Reply:
updated